pacai/util/mazeGenerator.py

```diff
@@ -77,7 +78,7 @@ def __str__(self):
 
         return s[:-1]
 
-    def add_wall(self, i, gaps=1, vert=True):
+    def add_wall(self, i, gaps=1, vert=True, rng=None):
```

Comment:
In all these functions, `rng` cannot be None (it will cause a crash).
Make `rng` a required parameter.